### PR TITLE
Fixed type hint for an argument of :class:`~.MoveAlongPath`

### PR DESCRIPTION
**Pull request by Kiran-Raj-Dev**
## Reviewer Checklist
- [ ] The PR title is descriptive enough for the changelog, and the PR is labeled correctly
- [ ] If applicable: newly added non-private functions and classes have a docstring including a short summary and a PARAMETERS section
- [ ] If applicable: newly added functions and classes are tested

### CODE REVIEW

**Comment by Kiran-Raj-Dev** [comment]
I messed up something, didn't I?

File: manim/animation/movement.py [hunk 1]
```diff
@@ -131,7 +131,7 @@ def construct(self):
     def __init__(
         self,
         mobject: "Mobject",
-        path: np.ndarray,
+        path: VMobject,
```
Comment (behackl) [inline manim/animation/movement.py:134]: Either this is rewritten as a string, or a separate import to VMobject is added at the top of the file. 

```suggestion
        path: "VMobject",
```

**Review by behackl — CHANGES_REQUESTED** [review]
Hey! This causes an unresolved import, but it is not hard to fix.

File: manim/animation/movement.py (hunk 1 above)
Comment (Kiran-Raj-Dev) [reply manim/animation/movement.py:134]: Ah! I see

How dod you figure?

Comment (behackl) [reply manim/animation/movement.py]: It is not a particularly uncommon mistake when working with type hints. The problem has to be with the line you changed, and you can see that the pipeline complains about not knowing `VMobject`: https://github.com/ManimCommunity/manim/runs/4548003604?check_suite_focus=true

Comment (Kiran-Raj-Dev) [reply manim/animation/movement.py]: Aha! I see. Thanks for clearing!

**Review by behackl — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**